### PR TITLE
enb/make: add coverage

### PR DESCRIPTION
**Pull request by zxqfox**

### CODE REVIEW

File: .enb/make.js [hunk 1]
```diff
@@ -18,13 +18,15 @@ module.exports = function(config) {
...
                 engines : {
                     bh : {
-                        tech : 'enb-bh/techs/bh-server',
+                        tech : 'enb-bh/techs/bh-server-include',
                         options : {
                             jsAttrName : 'data-bem',
                             jsAttrScheme : 'json'
-                        }
+                        },
+                        sourcemaps : true
```
Comment (blond) [inline .enb/make.js]: sourcemaps -> sourcemap

Надо перенести в поле `options` выше.

File: .travis.yml [hunk 2]
```diff
@@ -5,3 +5,8 @@ node_js:
 
 matrix:
   fast_finish: true
+
+after_success:
+  - |
+      npm install coveralls istanbul
```
Comment (remnev) [inline .travis.yml]: `istanbul` is unnecessary, isn't it?

File: .travis.yml [hunk 3]
```diff
@@ -5,3 +5,8 @@ node_js:
...
+      cat coverage/lcov.info | node_modules/coveralls/bin/coveralls.js && echo "Coverage data was sent to coveralls!"
```
Comment (remnev) [inline .travis.yml]: `node_modules/coveralls/bin/coveralls.js` —> `node_modules/.bin/coveralls`

**Comment by remnev** [comment]
Can you fix things with my comments and rebase from master?

File: .travis.yml (hunk 2 above)
Comment (zxqfox) [inline .travis.yml]: yup

**Comment by zxqfox** [comment]
Yup, will do it little bit later.

**Comment by remnev** [comment]
:+1:  merge it

**Pull request closed without merging**